ansible-playbook/requirements.txt

```diff
@@ -0,0 +1,2 @@
+openstacksdk
```

Comment:
Could we have a comment explaining which ansible features these are required for?

Reply:
This is an enabler for all the OpenStack modules at
https://docs.ansible.com/ansible/latest/modules/list_of_cloud_modules.html#openstack
I have been using os_server to create VMs. Where did you want the docs?

Reply:
A comment above the requirements would be grand.